Utils/Dataflow/055_documents2TTL/documents2ttl.py

```diff
@@ -299,11 +353,14 @@ def glance_parameter_extraction(param_name, json_data):
 
 
 def cds_parameter_extraction(param_name, json_data):
-    """
-    Extracting parameters from json string with CDS parameters
-    :param param_name: name of parameter, defined in *_CDS_ATTRS dict
-    :param json_data: json string with CDS parameters
-    :return:
+    """ Extract CDS parameter value from JSON.
+
+    :param param_name: name of the parameter, defined in *_CDS_ATTRS dict
+    :type param_name: str
+    :param json_data: 'CDS' part of the initial JSON
```

Comment:
```suggestion
    :param json_data: 'CDS' part of the initial JSON
    :type json_data: dict
```

*NOTE: I am not sure how this kind of suggestion will work, so maybe it would be safer to make a corresponding commit by hands.*

Reply:
Thanks, I missed this. Done in 2740e63.